jenkins/scripts/clean_resources.sh

```diff
@@ -11,7 +11,7 @@ set -eu
 # Usage:
 #  clean_resources.sh
 #
-CLIENT_VERSION="7.0.0"
+CLIENT_VERSION="v7.0.0"
```

Comment:
Either syntax, 7.0.0 or v7.0.0 works. The version listed in PyPI is 7.0.0 so I'd use that.